README.md

```diff
@@ -73,14 +73,13 @@ App management: Please read [issue #15807](https://github.com/appium/appium/issu
 
 ## NuGet
 
-NuGet Package: [](http://www.nuget.org/packages/Appium.WebDriver/)
+[NuGet Package](http://www.nuget.org/packages/Appium.WebDriver/)
 
 Dependencies:
 
 - [Selenium.WebDriver](http://www.nuget.org/packages/Selenium.WebDriver/)
-- [Newtonsoft.Json](http://www.nuget.org/packages/Newtonsoft.Json/)
-- [Selenium.Support](https://www.nuget.org/packages/Selenium.Support/)
-- [Castle.Core](https://www.nuget.org/packages/Castle.Core/)
+- [System.Text.Json](https://www.nuget.org/packages/System.Text.Json/)
```

Comment:
Do we need this here?  As this project, this deps is maybe for testing only https://github.com/appium/dotnet-client/blob/dfb65ac18916c94ba4ed1f3b0072c121c7fe826b/test/integration/Appium.Net.Integration.Tests.csproj#L19

I'm wondering if this deps can have a link to https://github.com/appium/dotnet-client/blob/main/src/Appium.Net/Appium.Net.csproj simply for product code, and test code is dotnet-client/test/integration/Appium.Net.Integration.Tests.csproj instead of listening all of them here

Reply:
you are right, removed `System.Text.Json`.
Regarding the link, I can add the following link which will show the NuGet dependencies
https://www.nuget.org/packages/Appium.WebDriver/absoluteLatest#dependencies-body-tab

Reply:
@kazucocoa, so what's your call, Add the link to NuGet dependencies tab?

Reply:
Either is fine. I mean maintaining the deps here as well, or leaving the link only so that we don't need to keep reflecting the deps